hhs_hosp/tests/test_run.py

```diff
@@ -89,40 +89,51 @@ def test_make_signal():
     with pytest.raises(Exception):
         make_signal(data, "zig")
 
-def test_pop_proportion():
```

Comment:
Note on this change, since the diffs are not helping with clarity here.

test_pop_proportion used to test two individual states as two separate asserts.

The replacement, test_transform_signal_pop, tests two **_geos_**, with both states in the first assert, and nation as the second assert.